catalog/opensearch/catalog-opensearch-source/src/main/java/ddf/catalog/source/opensearch/OpenSearchSource.java

```diff
@@ -212,8 +222,7 @@ public boolean isAvailable() {
                 return false;
             }
 
-            if (response != null && !(response.getStatus() >= 404 || response.getStatus() == 400
-                    || response.getStatus() == 402)) {
+            if (response != null && !(response.getStatus() >= 404 || response.getStatus() == 402)) {
```

Comment:
Since the WebClient is hitting the provided base url without any query parameters, it can still return a 400 and technically be available.  This conditional was relaxed to support this case.